connector/mqtt/vernemq/v2k-brigde/Dockerfile

```diff
@@ -1,26 +1,49 @@
-FROM node:10.16.3-alpine
+#
+# ---- Base Image ----
+FROM node:10.16.3-alpine AS base
 
 WORKDIR /opt/v2k_bridge
 
-
-RUN apk add --no-cache g++ \
-      make \
+RUN apk --no-cache add \
       bash \
-      python \
+      ca-certificates \
+      cyrus-sasl-dev \
+      g++ \
       lz4-dev \
+      make \
       musl-dev \
-      ca-certificates \
       openssl-dev \
-      zlib-dev \
-      openssl \
-      curl \
-      jq \
-      zstd
+      python
+
+RUN apk add --no-cache --virtual .build-deps gcc zlib-dev libc-dev bsd-compat-headers py-setuptools bash
 
 COPY package.json .
 COPY package-lock.json .
 
+#
+# ---- Install dependencies
 RUN npm install
-COPY . .
 
-ENTRYPOINT ["./bin/entryPoint.sh"]
\ No newline at end of file
+COPY ./bin/ ./bin
+COPY ./app ./app
+
+#
+# --- Production Image
+FROM node:10.16.3-alpine
+
+WORKDIR /opt/v2k_bridge
+ENV BASE_DIR="/opt/v2k_bridge"
+
+RUN apk --no-cache add \
```

Comment:
They are not required, I changed them by libsasl and lz4-libs, because they are required by node-rdkafka.

When removing them, this error shows
Error loading shared library libsasl2.so.3: No such file or directory (needed by /opt/v2k_bridge/node_modules/node-rdkafka/build/Release/../deps/librdkafka.so.1)
Error: Error loading shared library libsasl2.so.3: No such file or directory (needed by /opt/v2k_bridge/node_modules/node-rdkafka/build/Release/../deps/librdkafka.so.1)